common/src/test/java/org/apache/celeborn/common/network/ssl/SslConnectivitySuiteJ.java

```diff
@@ -185,6 +186,23 @@ public void testNormalConnectivityWorks() throws Exception {
         Function.identity());
   }
 
+  @Test
+  public void testNormalConnectivityWithAuthWorks() throws Exception {
```

Comment:
This test is strictly not related to this PR - but is one of the reasons why TLS support was introduced - to secure when auth is enabled.